doc/advanced-configuration/release-options.md

```diff
@@ -0,0 +1,226 @@
+
+# vm.args options
+
+These options are inserted into the `rel/files/vm.args` template.
+
+## node_name
+
+* **Type:** parameter
+* **Option:** value of `-sname` in [vm.args](../../advanced-configuration#options)
+* **Syntax:** Erlang node name: `name@host`
+* **Example:** `{node_name, "mongooseim@localhost"}.`
+
+## highload_vm_args
+
+* **Type:** block
+* **Option:** arguments in [vm.args](../../advanced-configuration#options): `+K`, `+A`, `+P`, `-env ERL_MAX_PORTS`
+* **Syntax:** command-line arguments
+* **Example:** `{highload_vm_args, "+P 10000000 -env ERL_MAX_PORTS 250000"}.`
+
+# TOML Options
+
+These options are inserted into the `rel/files/mongooseim.toml` template.
+
+## hosts
+
+* **Type:** parameter
+* **Option:** [`general.hosts`](../../advanced-configuration/general#generalhosts)
+* **Syntax:** comma-separated list of strings
+* **Example:** `{hosts, "\"localhost\", \"domain2\""}.`
+
+## host_config
+
+* **Type:** block
+* **Option:** [`host_config`](../../advanced-configuration/host_config.md)
+* **Syntax:** TOML block, one or more `[[host_config]]` sections.
+* **Example:**
+
+```
+{host_config, "
+[[host_config]]
+  host = \"anonymous.localhost\"
+
+  [host_config.auth]
+    methods = [\"anonymous\"]
+"}.
+```
+
+## auth_ldap
+
+* **Type:** block
+* **Option:** [`auth.ldap`](../../authentication-methods/ldap.md)
+* **Syntax:** TOML block, the `[auth.ldap]` subsection
+* **Example:**
+
+```
+{auth_ldap, "
+  [auth.ldap]
+    base = \"ou=Users,dc=esl,dc=com\"
+    filter = \"(objectClass=inetOrgPerson)\"
+"}.
+```
+
+## all_metrics_are_global
+
+* **Type:** parameter
+* **Option:** [`general.all_metrics_are_global`](../../advanced-configuration/general#generalall_metrics_are_global)
+* **Syntax:** boolean
+* **Example:** `{all_metrics_are_global, "false"}.`
+
+## s2s_addr
+
+* **Type:** block
+* **Option:** [`auth.s2s.address`](../../advanced-configuration/s2s#s2saddress)
+* **Syntax:** TOML key-value pair with the `address` option
+* **Example:**
+
+```
+{s2s_addr, "
+  address = [
+    {host = \"my.xmpp.org\", ip_address = \"192.0.100.1\"},
+    {host = \"your.xmpp.org\", ip_address = \"192.0.1.100\", port = 5271}
+  ]
+"}.
+```
+
+## s2s_default_policy
+
+* **Type:** parameter
+* **Option:** [`s2s.default_policy`](../../advanced-configuration/s2s#s2sdefault_policy)
+* **Syntax:** string
+* **Example:** `{s2s_default_policy, "\"deny\""}.`
+
+## outgoing_s2s_port
+
+* **Type:** parameter
+* **Option:** [`s2s.outgoing.port`](../../advanced-configuration/s2s#s2soutgoingport)
+* **Syntax:** integer
+* **Example:** `{s2s_outgoing_port, "5269"}.`
+
+## c2s_port
+
+* **Type:** parameter
+* **Option:** [`listen.c2s.port`](../../advanced-configuration/listen#listenport)
+* **Syntax:** integer
+* **Example:** `{c2s_port, "5222"}.`
+
+## s2s_port
+
+* **Type:** parameter
+* **Option:** [`listen.s2s.port`](../../advanced-configuration/listen#listenport)
+* **Syntax:** integer
+* **Example:** `{s2s_port, "5269"}.`
+
+## cowboy_port
```

Comment:
Do we keep `cowboy` here or should it be `http_port`?

Reply:
I wouldn't want to change this as we use it in the template variables.